desktop/assets/texts/pt-BR/preferences_dialog.ftl

```diff
@@ -17,8 +17,16 @@ log-filename-pattern-with-timestamp = Com data e hora
 storage-backend = Localização dos arquivos de save
 storage-backend-disk = Disco
 storage-backend-memory = Memória
+recent-limit = Limite de recentes
 recent-clear = Limpar
 theme = Tema
 theme-system = Padrão do sistema
 theme-light = Claro
 theme-dark = Escuro
+# See for context https://github.com/FeralInteractive/gamemode
+gamemode = Modo de jogo
```

Comment:
I don't think this should be translated

Reply:
I'm not touching any context anymore :smile:

The context already mentions it's a name of software, the question is whether adding further context will help

Reply:
This got undone since on CrowdIn.